test/temp-sensor/Test.cs

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace temp_sensor
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using common;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public class Test
```

Comment:
I expect each test file/class represent 1 test like tempSensor. is it correct?  If so, how these tests be called from main program dynamically; expect new tests will be added in the future.

Reply:
The high-level thinking is that each test will be it's own app, unless we introduce some kind of test framework. But I'll know more when I introduce a 2nd test.

Reply:
we can discuss more when working on 2nd test.